code/model/modelinterp.cpp

```diff
@@ -2613,7 +2613,7 @@ void interp_configure_vertex_buffers(polymodel *pm, int mn)
 
 	int time_elapsed = timer_get_milliseconds() - milliseconds;
 
-	mprintf(("BSP Parse took %d milliseconds.", time_elapsed));
+	mprintf(("BSP Parse took %d milliseconds.\n", time_elapsed));
```

Comment:
This could very well be hidden behind an nprintf filter. It's information that is not generally useful, and causes quite a bit of clutter if large models are being loaded.